src/main/java/com/wcc/platform/service/CmsService.java

```diff
@@ -41,12 +42,17 @@ public CmsService(final ObjectMapper objectMapper, final PageRepository pageRepo
    * @return Pojo TeamPage.
    */
   public TeamPage getTeam() {
-    try {
-      return objectMapper.readValue(
-          FileUtil.readFileAsString(PageType.TEAM.getFileName()), TeamPage.class);
-    } catch (JsonProcessingException e) {
-      throw new PlatformInternalException(e.getMessage(), e);
+    final var page = pageRepository.findById(PageType.TEAM.getPageId());
+
+    if (page.isPresent()) {
```

Comment:
Should we have a fallback to the json file ? or we are confident that we will be fine to deploy SurrealDB instance without any issue ?

Reply:
we are on dev mode still, also I don't think we should keep 2 source of true. In that case I think we should return not found page and not show if it do not exist.